tests/rustdoc-gui/sidebar-resize-close-popover.goml

```diff
@@ -0,0 +1,20 @@
+// Checks sidebar resizing close the Settings popover
```

Comment:
It was already doing that (because of the blur event). That's still a good thing to test though. However, we need to check for the "i" popups (codeblocks and important traits).

Reply:
Firefox.

Reply:
I just re-checked and indeed, it's not. Maybe my memory is lying. ^^'